test/functions/lambda/lambdaCapturing.good

```diff
@@ -1,3 +0,0 @@
-lambdaCapturing.chpl:5: In function 'test':
```

Comment:
It looks like you deleted the good file, but not the source file here.

Reply:
I'm not seeing that file now so I must have removed it at some point.